diff --git a/node_modules/@bablr/regex-vm/lib/internal/literals.js b/node_modules/@bablr/regex-vm/lib/internal/literals.js
index bbd1187..2da1a50 100644
--- a/node_modules/@bablr/regex-vm/lib/internal/literals.js
+++ b/node_modules/@bablr/regex-vm/lib/internal/literals.js
@@ -61,7 +61,7 @@ export const testAny = (c) => c !== sym.gap;
 export const getCharTester = (node, flags) => {
   if (flags.ignoreCase) {
     let value = upperCode(node);
-    return (c) => upperCode(String.fromCharCode(c)) === value;
+    return (c) => upperCode(String.fromCodePoint(c)) === value;
   } else {
     let expected = code(node);
     return (c) => c === expected;
@@ -80,7 +80,7 @@ export const getCharClassRangeTester = (node, flags) => {
   if (flags.ignoreCase) {
     let minCode = upperCode(min);
     let maxCode = upperCode(max);
-    return (c) => inRange(upperCode(String.fromCharCode(c)), minCode, maxCode);
+    return (c) => inRange(upperCode(String.fromCodePoint(c)), minCode, maxCode);
   } else {
     let minCode = code(min);
     let maxCode = code(max);
diff --git a/node_modules/@bablr/regex-vm/lib/internal/regex.js b/node_modules/@bablr/regex-vm/lib/internal/regex.js
index d95a290..49ea718 100644
--- a/node_modules/@bablr/regex-vm/lib/internal/regex.js
+++ b/node_modules/@bablr/regex-vm/lib/internal/regex.js
@@ -42,7 +42,7 @@ let character = (value, flags) => (next) => {
     next,
     match: (state, { value }) => {
       if (tester(value)) {
-        state.result = state.result.push(value === sym.gap ? null : String.fromCharCode(value));
+        state.result = state.result.push(value === sym.gap ? null : String.fromCodePoint(value));
         return next;
       } else {
         return null;
